### PR TITLE
Document the <frameworkReferences> node

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/1874

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[7c07729](https://github.com/NuGet/docs.microsoft.com-nuget/commits/7c07729b9ca1607a2aacb68e57f0bfebd1083d74)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-frameworkReference/docs/reference/nuspec.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1917) | [Details](#user-content-8224f5a0-19c7b1d3)

<a id="8224f5a0-19c7b1d3"></a>
### [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-frameworkReference/docs/reference/nuspec.md)
 - **Line 232**: **[Warning]** `````Cannot find bookmark '#specifying-explicit-assembly-references' in 'reference/nuspec.md'`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281816527086-1917%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=oVZSmGeKVXUY7IVbvGHkfswde7fEgoNpyWVK4YTbF6M%3D&st=2020-02-28T18%3A12%3A33Z&se=2020-03-30T18%3A17%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/nuspec.md [hunk 1]
```diff
@@ -347,39 +349,7 @@ The following lines indicate dependencies on the same packages, but specify to i
...
-    </group>
-
-    <group targetFramework="sl30">
-    </group>
-</dependencies>
-```
-
-<a name="specifying-explicit-assembly-references"></a>
-
+F
```
Comment (zkat) [inline docs/reference/nuspec.md]: Why is there a random floating F? 😂

File: docs/reference/nuspec.md [hunk 2]
```diff
@@ -739,6 +709,33 @@ Empty folders can use `.` to opt out of providing content for certain combinatio
...
+Framework References are a .NET Core concept representing shared frameworks such as WPF or Windows Forms.
+By specifying a shared framework, the package ensures that all its framework dependencies are included in the referencing project.
+
+Each `<group>` element requires a `targetFramework` attribute and zero or more `<frameworkReference>` elements.
+
+The following example shows a nuspec generated for a .NET Core WPF project.
+Note that hand authoring nuspecs that contain framework references is not recommended. Consider using the [targets](msbuild-targets.md) pack instead, which will automatically infer them from the project.
+
+```xml
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
```
Comment (zkat) [inline docs/reference/nuspec.md]: How can I check that this is a valid schema?

Comment (nkolev92) [reply docs/reference/nuspec.md]: It's not uploaded anywhere...I just used the nuspec that was generated by dotnet pack.

File: docs/reference/nuspec.md (hunk 1 above)
Comment (nkolev92) [reply docs/reference/nuspec.md]: Just wanted to make sure the reviewers are paying attenion :D 

Will revert these changes

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[11d8ddf](https://github.com/NuGet/docs.microsoft.com-nuget/commits/11d8ddfe913a3bd7dcdbc6e61af81aa94a8de366)_: 

:clock10: **Pending**: waiting for processors ([5](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 5 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[11d8ddf](https://github.com/NuGet/docs.microsoft.com-nuget/commits/11d8ddfe913a3bd7dcdbc6e61af81aa94a8de366)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-frameworkReference/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1917) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281850241453-1917%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3W9WrzdVN8aJO4%2FnBYzgP6VhITF5p8HZom7Q7lBXMFo%3D&st=2020-02-28T18%3A46%3A00Z&se=2020-03-30T18%3A51%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[3c51428](https://github.com/NuGet/docs.microsoft.com-nuget/commits/3c514284ac5bbe42103d98766aff816597cc1339)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-frameworkReference/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1917) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281852233622-1917%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=aavWsJbh2kAgMmGOeL3RbzpYYWntVZSuuUv0vhQOC%2Bk%3D&st=2020-02-28T18%3A48%3A02Z&se=2020-03-30T18%3A53%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**